dockerhub-script/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Run DockerHub Script
+
+description: |
+  This composite action generates a DockerHub token and runs a script with it.
+
+inputs:
+  GPUCIBOT_DOCKERHUB_USER:
```

Comment:
I think this should be a more generic name here since we've dropped the `gpuci` moniker everywhere. Probably just drop the `GPUCIBOT_` prefix?